dask/array/tests/test_ufunc.py

```diff
@@ -394,7 +394,7 @@ def test_array_ufunc():
     x = np.arange(24).reshape((4, 6))
     d = da.from_array(x, chunks=(2, 3))
 
-    for func in [np.sin, np.isreal, np.sum, np.negative, partial(np.prod, axis=0)]:
+    for func in [np.sin, np.sum, np.negative, partial(np.prod, axis=0)]:
```

Comment:
Why is removing `np.isreal` test necessary?

Reply:
It's tested higher up - the confusing thing here was that this whole file is about ufuncs, and `isreal` isn't a ufunc. It probably should be moved.

The test wasn't broken though, can put it back if you prefer.

Reply:
No need, if it's already tested, that's fine by me. Was just checking, thanks for clarifying.